scipy/optimize/_highs/src/libhighs_export.h

```diff
@@ -0,0 +1,50 @@
+
```

Comment:
New platform-specific generated file.  This reproduction works on all targeted platforms we have tested